lumapps/api/client.py

```diff
@@ -356,7 +369,7 @@ def upload_call(self, fpath: Path, metadata: dict, *name_parts, **params):
         resp.raise_for_status()
         return resp.json()
 
-    def get_call(self, *name_parts, **params):
+    def get_call(self, *name_parts, cursor=None, **params):
```

Comment:
Please make this happen using what already exists, ie parameters or payloads.

Reply:
You're right, done it.